### ISSUE TITLE
Cannot find controller for resource type: version=v1beta2 and kind=KafkaPrincipalRole

### ISSUE THREAD

**Issue by Akash-Mair** [streamthoughts/jikkou#190]
Hi!

Thanks for the help and continued support :) 
I'm running version `0.19.0` and when running the following config, getting the above error:

```yaml
---
apiVersion: "kafka.jikkou.io/v1beta2"
kind: KafkaPrincipalRole
metadata:
  name: "Read"
spec:
  acls:
    - resource:
        type: 'topic'
        pattern: 'name'
        patternType: 'LITERAL'
      type: "ALLOW"
      operations: [ 'WRITE' ]
      host: "*"
---
apiVersion: "kafka.jikkou.io/v1beta2"
kind: KafkaPrincipalAuthorization
metadata:
  name: 'User:me'
spec:
  roles:
    - "Read"
```

I did try base this sample from the examples in the repo - let me know if I've made a mistake 
Many thanks!

**Comment by Akash-Mair** [comment]
Alternatively this seems to work:

```yaml
apiVersion: "kafka.jikkou.io/v1beta2"
kind: KafkaPrincipalAuthorization
metadata:
  name: 'User:me'
spec:
  acls:
    - resource:
        type: 'topic'
        pattern: 'name'
        patternType: 'LITERAL'
      type: "ALLOW"
      operations: [ 'WRITE' ]
      host: "*"
```

**Comment by fhussonnois** [comment]
Hi @Akash-Mair, the resource kind has changed is the latests version. Please find the documentation here : https://streamthoughts.github.io/jikkou/docs/user-guide/kafka/resources/acls/

**Comment by dani** [comment]
I have the same error with the following
```
---
apiVersion: kafka.jikkou.io/v1beta2
kind: KafkaPrincipalRole
metadata:
  name: article-consumer
spec:
  acls:
    - type: ALLOW
      operations: [READ, DESCRIBE]
      resource:
        type: TOPIC
        pattern: article-associations
        patternType: LITERAL

    - type: ALLOW
      operations: [READ]
      resource:
        type: GROUP
        pattern: article-associations
        patternType: LITERAL
```
Can't find in the online doc what kind should it be instead

**Comment by fhussonnois** [comment]
Hi @dani, thank you for reporting the problem. This issue was fixed in Jikkou 0.20.0.
